src/components/displayCategories.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
-import { useDispatch } from 'react-redux';
-import { checkBookStatus } from '../redux/Categories/Categories';
+import { useDispatch, useSelector } from 'react-redux';
+// import './styles/Categories.css';
```

Comment:
OPTIONAL
_I make it optional but this is a general requirement for every project._
- [ ] To make clean code kindly delete commented code.